pkg/backends/tcp.go

```diff
@@ -210,8 +210,8 @@ func (ns *TCPSession) Close() error {
 // Command line
 // **************************************************************************
 
-// TCPListenerCfg is the cmdline configuration object for a TCP listener
-type TCPListenerCfg struct {
+// tCPListenerCfg is the cmdline configuration object for a TCP listener
+type tCPListenerCfg struct {
```

Comment:
Is this really the canonical way to name these variables in Go? It looks so strange...

Reply:
What about `tcpListenerCfg`?

Reply:
You mean with the small first letter?  Yes, I agree it looks deeply weird.  But that's how you do it in Go.  A small first letter means private, capital first letter means public.

Reply:
The linter is usually very opinionated about these things, but doesn't complain about `tcpListenerCfg`.  I don't care, I just don't want linter errors.  😁

I'll change them to that style.